app/models/membership_application.rb

```diff
@@ -41,6 +41,9 @@ class MembershipApplication < ApplicationRecord
 
   scope :open, -> { where.not(state: [:accepted, :rejected]) }
 
+  delegate :full_name, to: :user, prefix: true, allow_nil: true
+  delegate :email, to: :user, prefix: true, allow_nil: true
+
```

Comment:
@RobertCram @weedySeaDragon keeping `#full_name` delegate and removing `#email` delegate